canvas/templates/canvas/problem_set_snippet.html

```diff
@@ -1,11 +1,17 @@
+{% load canvas %}
+{% tokens_column_name event as tokens_column_name %}
```

Comment:
can you please move this close to where you used it? It will be hard to understand what tokens_column_name is when we see this in the future.